assets/js/common/Input/Textarea.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+
+function Textarea({
+  id,
+  name,
+  value,
+  initialValue,
+  placeholder,
+  disabled = false,
+  onChange,
+}) {
+  const [initialInputValue, setValue] = useState(initialValue);
+  const defaultOnChange = (e) => setValue(e.target.value);
```

Comment:
Why this? `value` wasn't enough?

Reply:
As mentioned privately the reasons are two:
- I got inspired by the api that also rc-input exposes
- without `initialValue` and `defaultOnChange` we can only have value controlled from the outside

I don't mind here either, whatever you prefer.

Reply:
Leave it, no problem with this :+1: